nevergrad/optimization/mutations.py

```diff
@@ -66,26 +66,26 @@ def portfolio_discrete_mutation(self, parent: tp.ArrayLike, intensity: tp.Option
             intensity = 1 if dimension == 1 else int(self.random_state.randint(1, dimension))
         if dimension == 1:  # corner case.
             return self.random_state.normal(0., 1., size=1)  # type: ignore
-        boolean_vector = [True for _ in parent]
+        boolean_vector = np.array(np.ones(dimension), dtype=bool)
```

Comment:
```suggestion
        boolean_vector = np.ones(dimension, dtype=bool)
```

Reply:
How I could come up with something THAT stupid ...